datafusion/physical-optimizer/Cargo.toml

```diff
@@ -49,6 +49,7 @@ datafusion-physical-plan = { workspace = true }
 itertools = { workspace = true }
 log = { workspace = true }
 recursive = { workspace = true, optional = true }
+tokio = { workspace = true }
```

Comment:
@zhuqi-lucas I think this could be moved to dev-dependencies since the new code only uses it in tests

Reply:
Thank you @simonvandel , good point, addressed in latest PR.